adal/src/main/java/com/microsoft/aad/adal/Discovery.java

```diff
@@ -152,7 +153,7 @@ private boolean queryInstance(final URL authorizationEndpointUrl) {
         // It will query prod instance to verify the authority
         // construct query string for this instance
         URL queryUrl;
-        boolean result = false;       
+        boolean result;
```

Comment:
compile fails if this is made final.

---
In reply to: [65640803](https://github.com/AzureAD/azure-activedirectory-library-for-android/pull/652#discussion_r65640803) [](ancestors = 65640803)